### PR TITLE
STYLING

### PR DESCRIPTION
**Pull request by emmanuellmhango**
In this PR, I have implemented the following:

- Styling the NavBar
- Styling the books
- Styling the form

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -1,20 +1,66 @@
...
+        <div className="category">{category}</div>
+        <div className="title">{title}</div>
+        <div className="author">{author}</div>
+        <ul className="book-footer-links">
+          <li><NavLink className="footer-link">Comments</NavLink></li>
+          <li className="space-li"> | </li>
+          <li><NavLink className="footer-link" onClick={delBook}>Remove</NavLink></li>
+          <li className="space-li"> | </li>
+          <li><NavLink className="footer-link">Edit</NavLink></li>
+        </ul>
```
Comment (danmainah) [inline src/components/Book.js:27]: (sorry for applying the review comment in the wrong section)
- [ ] @emmanuellmhango great work on your styling, however, check on your navbar the arrangement differs from the design given slightly :

Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @emmanuellmhango , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/Book.js (hunk 1 above)
Comment (Whoistolu) [reply src/components/Book.js:27]: Well resolved 👍🏽

Comment (Whoistolu) [reply src/components/Book.js:27]: - [ ] [REQUIRED] Well done @emmanuellmhango , I see that the changes you made was just changing the color. Kindly note that the last reviewer recommended that you change the arrangement of the content in your project navbar. I have checked your design and found this:
 
Screenshot 2022-12-16 at 22 08 29

Kindly restyle it to be like the Zeplin design

**Review by Whoistolu — CHANGES_REQUESTED** [review]
### Hi @emmanuellmhango 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Book.js (hunk 1 above)
Comment (Whoistolu) [reply src/components/Book.js:27]: Well resolved 👍🏽

Comment (emmanuellmhango) [reply src/components/Book.js:27]: Thank you. @Whoistolu 

navbar

Comment (Whoistolu) [reply src/components/Book.js:27]: Great implementation @emmanuellmhango !

**Pull request merged**